resources/application-ide.properties

```diff
@@ -6,5 +6,6 @@ identity.store.alias=cid
 identity.key.password=changeit
 trust.keystore=resources/truststore.jks
 trust.store.password=changeit
-data.key.service.url=https://localhost:8443
+data.key.service.url=http://localhost:8080
```

Comment:
Why are we changing to http? Is this properties file used for the tests...wouldn't we want https to be part of it still?

Reply:
Hi Steve - this is only for when it's running locally in your ide - not for tests.